index.html

```diff
@@ -6,21 +6,135 @@
     <title>Dashboard</title>
     <link rel="stylesheet" href="./styles/common.css">
     <link rel="stylesheet" href="./styles/dashboard.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto+Slab:wght@100..900&display=swap" rel="stylesheet">
 </head>
 <body>
    
-    <div class="wrapper">
-        <div class="sidebar">
-            <ul>
-                <li>Meals</li>
-                <li>Report</li>
-                <li>Stats</li>
-            </ul>
+    <div class="navbar">
+        <div class="home-button">
+            <a href="../index.html"><button><img src="../images/dashboard_active.png" alt="dashboard" height="50px"></button></a>
+        </div>
+        <div class="menu-items">
+            <a href="./pages/diary.html"><button class="navbar-button">Diary</button></a>
+            <a href="./pages/meals.html"><button class="navbar-button">Meals</button></a>
+            <a href="./pages/recipes.html"><button class="navbar-button">Recipes</button></a>
+            <a href="./pages/stats.html"><button class="navbar-button">Stats</button></a>
         </div>
+    </div>
+
+    <div class="wrapper">
+
         <div class="content">
-            Dashboard
-            <a href="./pages/meals.html"> Go to meals</a>
+            <div class="dashboard-hero">
+                <div class="welcome-message">Welcome back, Boki</div>
+                <div class="user-image">
+                    <img src="./images/profile.jpeg" alt="Profile picture">
+                </div>
+                <div class="user-streak">
+                    <ul>
+                        <li class="active-streak">M</li>
```

Comment:
fyi: instead of `active-streak` and `inactive-streak` i would be better just have
streak (means regular one) and streak-active. Usually, we put modificator in the end.

changes not required. Just fyi